### PR TITLE
Add Ryn's Farms

### PR DESCRIPTION
**Pull request by loicreynier**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -18868,6 +18868,30 @@ plugins:
...
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Riften Watchtower Nav Fix.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Ryn''s Standing Stones](https://www.nexusmods.com/skyrimspecialedition/mods/64969/)'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s_Standing_Stones_Ryn''s_Farms_PATCH.esp")'
+    clean:
+      - crc: 0x1BE50A99
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164/)'
```
Comment (sibir-ine) [inline masterlist.yaml]: Clean checksums don't support Markdown, you can just copy & paste the report xEdit generates without touching it:

```suggestion
      - crc: 0x1BE50A99
        util: 'SSEEdit v4.0.4'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -18868,6 +18868,30 @@ plugins:
...
+  - name: 'Ryn''s Farms.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72305/'
+        name: 'Ryn''s Farms'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Sewers'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]: The patch messages should only be displayed if the patched mod is actually active:

```suggestion
        condition: 'active("SkyrimSewers.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -18868,6 +18868,30 @@ plugins:
...
+      - <<: *patch3rdParty
+        subs:
+          - 'Rift Watchtower NavFix'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Riften Watchtower Nav Fix.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        condition: 'active("RiftWatchtowerNavFix.esp") and not active("Ryn''s Farms - Riften Watchtower Nav Fix.esp")'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -18868,6 +18868,30 @@ plugins:
...
+      - <<: *patch3rdParty
+        subs:
+          - '[Ryn''s Standing Stones](https://www.nexusmods.com/skyrimspecialedition/mods/64969/)'
+          - '[Ryn''s Skyrim Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/73778/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s_Standing_Stones_Ryn''s_Farms_PATCH.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        condition: 'active("Ryn''s Standing Stones.esp") and not active("Ryn''s_Standing_Stones_Ryn''s_Farms_PATCH.esp")'
```

File: masterlist.yaml [hunk 5]
```diff
@@ -18868,6 +18868,30 @@ plugins:
...
+        name: 'Ryn''s Farms'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Sewers'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Rift Watchtower NavFix'
```
Comment (sibir-ine) [inline masterlist.yaml]: I'd still use the mod's name & then specify the package (or plugin if necessary) that the patch is for:

```suggestion
          - 'Navigator - Navmesh Fixes (Rift Watchtower NavFix)'
```

File: masterlist.yaml [hunk 6]
```diff
@@ -18868,6 +18868,30 @@ plugins:
...
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Riften Watchtower Nav Fix.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Ryn''s Standing Stones](https://www.nexusmods.com/skyrimspecialedition/mods/64969/)'
```
Comment (sibir-ine) [inline masterlist.yaml]: You don't need to link to the patched mod in a patch message; presumably users already know where they got it from:

```suggestion
          - 'Ryn''s Standing Stones'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@loicreynier, along with the suggestions I've made, the mod overhauls a number of existing farms, so the __Location - Overhauls__ would be more appropriate. The `patch3rdParty` should also be sorted alphabetically by patched mod name (i.e., _Navigator - Navmesh Fixes (Rift Watchtower NavFix)_ -> _Ryn's Standing Stones_ -> _Skyrim Sewers_).

**Comment by loicreynier** [comment]
@sibir-ine thanks for the review. I applied your changes and sorted the patches alphabetically.

**Comment by sibir-ine** [comment]
> I applied your changes and sorted the patches alphabetically.

Thanks, you just need to move the mod to the __Location - Overhauls__ section (alphabetically sorted, of course) & it'll be ready to merge.

**Comment by loicreynier** [comment]
Whoops forgot about that. Done.

**Comment by sibir-ine** [comment]
Almost, just needs to be moved between `Rorikstead.esp` & `Sanctuary Reborn.esp`.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**